File: Python/tests/test_igt_orl.py
import pytest

from hbayesdm.models import igt_orl


def test_igt_orl():
    _ = igt_orl(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_igt_orl.R
context("Test igt_orl")
library(hBayesDM)

test_that("Test igt_orl", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(igt_orl(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
